### PR TITLE
Create build_lammps_19Jan2022_ARCHER2_gcc_1020.md

### PR DESCRIPTION
**Pull request by jsindt**
Adding build instructions for LAMMPS on the full system.

### CODE REVIEW

File: apps/LAMMPS/build_lammps_19Jan2022_ARCHER2_gcc_1020.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Setup your environment
+----------------------
+
+Load the correct modules:
+
+   ```bash
+   module restore
+   module swap PrgEnv-cray PrgEnv-gnu 
+   module load cray-fftw module load cmake
```
Comment (aturner-epcc) [inline apps/LAMMPS/build_lammps_19Jan2022_ARCHER2_gcc_1020.md:21]: Typo on this line, should be either `module load cray-fftw cmake` or two lines

File: apps/LAMMPS/build_lammps_19Jan2022_ARCHER2_gcc_1020.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        -D PKG_ASPHERE=yes -D PKG_BODY=yes -D PKG_CLASS2=yes        \
+        -D PKG_COLLOID=yes -D PKG_COMPRESS=yes -D PKG_CORESHELL=yes \
+        -D PKG_DIPOLE=yes -D PKG_GRANULAR=yes -D PKG_MC=yes         \
+        -D PKG_MISC=yes -D PKG_KSPACE=yes -D PKG_MANYBODY=yes       \
+        -D PKG_MOLECULE=yes -D PKG_MPIIO=yes -D PKG_OPT=yes         \
+        -D PKG_PERI=yes -D PKG_QEQ=yes -D PKG_SHOCK=yes             \
+        -D PKG_SRD=yes -D PKG_RIGID=yes                             \
+        ../cmake/
+  make -j 8
+  ```
```
Comment (aturner-epcc) [inline apps/LAMMPS/build_lammps_19Jan2022_ARCHER2_gcc_1020.md:45]: May be useful to say where in the path this puts the executable and what the executable will be called.

**Review by aturner-epcc — CHANGES_REQUESTED** [review]
Just a couple of minor comments

File: apps/LAMMPS/build_lammps_19Jan2022_ARCHER2_gcc_1020.md (hunk 1 above)
Comment (jsindt) [reply apps/LAMMPS/build_lammps_19Jan2022_ARCHER2_gcc_1020.md]: Good spot -- fixed now.

File: apps/LAMMPS/build_lammps_19Jan2022_ARCHER2_gcc_1020.md (hunk 2 above)
Comment (jsindt) [reply apps/LAMMPS/build_lammps_19Jan2022_ARCHER2_gcc_1020.md:46]: Good point -- I've added a sentence to address this.

**Review by aturner-epcc — APPROVED** [review]
Thanks @jsindt

**Pull request merged**